store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9925,8 +9925,51 @@ TODO: delete them permanently from here
   <defaultCOSValue>FALSE</defaultCOSValue>
   <desc>Creates unsubscribe system folder</desc>
  </attr>
- <attr id="4004" name="zimbraPrefCalenderScaling" type="enum" value="10,15,30" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.1.0">
+<attr id="4004" name="zimbraPrefCalenderScaling" type="enum" value="10,15,30" cardinality="single" optionalIn="account,cos" flags="accountInherited,domainAdminModifiable" since="9.1.0">
   <defaultCOSValue>30</defaultCOSValue>
   <desc>Default calendar resolution preference</desc>
 </attr>
+<attr id="4007" name="zimbraBackupConfigEnabled" type="boolean" cardinality="single" optionalIn="globalConfig,domain,cos,account" flags="accountCosDomainInherited,domainInherited" since="9.1.0">
+    <desc>Whether or not account is eligible for backup</desc>
+  </attr>
+<attr id="4008" name="zimbraBackupEnabled" type="boolean" cardinality="single" optionalIn="globalConfig" since="9.1.0">
+    <globalConfigValue>TRUE</globalConfigValue>
+    <desc>If true, enables Backup globally and if there is pre-configured data the crontab should get that schedule</desc>
+  </attr>
+<attr id="4009" name="zimbraBackupSyncUsers" type="boolean" cardinality="single" optionalIn="globalConfig" since="9.1.0">
+    <globalConfigValue>FALSE</globalConfigValue>
+    <desc>If true,  will ensure the same users have backup and Dumpster enabled</desc>
+  </attr>
+<attr id="4010" name="zimbraBackupStartTime" type="astring" max="256" cardinality="single" optionalIn="globalConfig,server"  flags="serverInherited" since="9.1.0">
+    <globalConfigValue>02:00</globalConfigValue>
+    <desc>Starts backup each day at this time. This attribute is used for cronjob only to set hour</desc>
+  </attr>
+<attr id="4011" name="zimbraBackupRetentionDays" type="integer" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>8</globalConfigValue>
+    <desc>The number of days backups will be kept. Backup Scheduler deletes backups older than the set value</desc>
+  </attr>
+<attr id="4012" name="zimbraBackupBlobsCompress" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
+    <globalConfigValue>TRUE</globalConfigValue>
+    <desc>If True, Blobs inside backups are compressed in zip format by default unless specified</desc>
+  </attr>
+<attr id="4013" name="zimbraBackupBlobsCompressZipStore" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="9.1.0">
```

Comment:
Prefer `zimbraBackupBlobsCompressZipStoreEnabled` for boolean

Reply:
Removed this attribute.